### ISSUE TITLE
Small jump in vertical scroll on first filter-tap, chrome-android

### ISSUE THREAD

**Issue by 4kbt** [findthemasks/findthemasks#64]
There is a small jump in vertical scroll position on chrome-android the first time I tap on a filter after page load (new site, as of 14:06 Pacific , March 21). 

Steps to repro:

1. Open site (Pixel 3a, chrome 80.0, Android 10)
2. Scroll down to filters, with filters near bottom of screen
3. Tap a filter (FL works).

Expected:
Filter filters results below

Happens:
Vertical scroll jumps by ~0.5-1 screen upward
Filter filters results below.

**Comment by cjimmy** [comment]
I was able to reproduce this with a Samsung Note 10+ on Chrome.

**Comment by eZanmoto** [comment]
I can't reproduce the jump when clicking a filter, but I do notice a jump when the page first loads due to the Twitter styling being added. I've submitted a hacky PR that adds a fixed height to the Twitter row to resolve this, but it can add a bit of extraneous spacing because we can't know in advance what height we're going to need (unless those tweets are static, in which case we can hard-code it to a better approximation). I've used a device-independent unit, but this should still be ideally tested on different devices to make sure that no elements are overlapping and that the spacing isn't problematic.
